src/test/data/JsonSerializableGuestBookTest/typicalGuestGuestBook.json

```diff
@@ -62,6 +68,7 @@
     "dateRange": "01/01/23 - 07/01/23",
     "numberOfGuests": "3",
     "isRoomClean": "yes",
-    "bill": "1000.01"
+    "bill": "1000.01",
+    "request" : "Help me to fix Wi-Fi!"
```

Comment:
Should this be phrased in the perspective of a hotel manager? E.g "Fix Wifi", as this will be entered by the hotel manager him/herself

Reply:
changed